Commands/MessageCount/MessageCountSystem.go

```diff
@@ -5,13 +5,37 @@
 		-msgcount
 */}}
 
-{{$x:=dbIncr .User.ID "msgCount" 1}}{{$list:=""}}
-{{if reFind `\A-m(?:essages?|sg)?c(?:ount)?` .Message.Content}}
-	 You have typed {{$x}} messages.
-{{end}}
-{{if reFind `\A-t(?:op)?m(?:essages?|sg)?` .Message.Content}}
-	{{range dbTopEntries "msgCount" 10 0}}
-		{{- $list =print $list .User.Username " - " .Value "\n" -}}
-	{{end}}
-	{{sendMessage nil (cembed "title" "Message Count Leaderboard" "description" $list)}}
-{{end}}
+{{/* DONT TOUCH UNLESS YOU KNOW WHAT UR DOING c: */}}
+{{/* define variables */}}
+{{ $avatar := (joinStr "" "https://cdn.discordapp.com/avatars/" (toString .User.ID) "/" .User.Avatar ".png") }}
```

Comment:
unnecessary () and no need to toString the userID, use print REEEEE, also unnecessary variable cus you could just use it inside Line 12's embed via (sdict "url" $avatar) -> (sdict "url" (print "https://cdn.discordapp.com/avatars/" .User.ID "/" .User.Avatar ".png")), altho could be argued as a necessity for readability lol